content/java/concepts/calendar/terms/clear/clear.md

```diff
@@ -0,0 +1,68 @@
+---
+Title: '.clear()'
+Description: 'A method designed to reset or clear specific fields of a Calendar instance.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Methods'
+  - 'Calendar'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`clear()`** method of the [`Calendar`](https://www.codecademy.com/resources/docs/java/calendar) class is used to reset or clear specific fields of the given instance to their default values. It allows the reset of individual fields, such as `YEAR`, `MONTH`, `DAY_OF_MONTH`, `HOUR`, etc., to their initial state.
```

Comment:
```suggestion
The **`.clear()`** method of the `Calendar` class is used to reset or clear specific fields of the given instance to their default values. It allows the reset of individual fields, such as `YEAR`, `MONTH`, `DAY_OF_MONTH`, `HOUR`, etc., to their initial state.
```
We generally don't link out to parent entries because the Docs navigation provides those automatically.